### DISCUSSION TITLE
macOS Safari 上，Doc 网站页面在图片全部加载出来之前，滑动时掉帧很严重

### DISCUSSION THREAD

**Discussion by LouisTay** [spencerwooo/onedrive-vercel-index#397] [Q&A]
macOS Safari 上，Doc 网站页面在图片全部加载出来之前，滑动时掉帧很严重，Chrome就没有这个问题。
Safari看的好难受，是浏览器的问题吗？

**Comment by LouisTay** [comment]
在macOS的Safari上，随便打开一个图片比较多的页面，比如[这个](https://onedrive-vercel-index.spencerwoo.com/zh/docs/showcase/file-preview)，在所有图片全部加载出来之前，滑动特别卡，等滑动到底部并且所有图片加载完成后，再滑动就不卡了。但是我用chrome试了一下好像就没有这个问题

**Comment by wc7086** [comment]
可能是用到了webkit没有的特性导致的

**Comment by spencerwooo** [comment]
@LouisTay 我貌似不能复现。

https://user-images.githubusercontent.com/32114380/153325932-73ebdd3d-41e2-4e2a-86fe-98b4650885d4.mp4

> - Safari Version 15.2 (17612.3.6.1.6)
> - MacBook Pro (14-inch, 2021) M1 Pro
> - macOS Monterey Version 12.1

**Reply by LouisTay** [reply]
我录了个屏，下面这个是 Safari 的，刚开始滑动的时候，图片没有全部加载完，滑动就感觉很卡，图片加载完的时候就很顺畅了：

https://user-images.githubusercontent.com/89720279/153337887-aa39271b-1a8c-4ddc-9dad-aebe140ffe56.mp4

然后我看 Chrome 在加载图片的时候也很顺畅：

https://user-images.githubusercontent.com/89720279/153337933-037f9fb4-b5a3-45cd-9720-aeb7643dbc9a.mp4


> MacBook Pro (15-inch, 2018) Intel Core i7
> macOS Monterey Version 12.2
> Safari 版本15.3 (17612.4.9.1.5)

Safari 重启、清空缓存都尝试了，还是这样。难道是 macOS 12.2 的问题？

**Reply by spencerwooo** [reply]
要不问问 nextra 那边？https://github.com/shuding/nextra
